### ISSUE TITLE
Error on Github Action

### ISSUE THREAD

**Issue by silex-fabrice** [patrickedqvist/wait-for-vercel-preview#22]
Hello,

After your last update on this github action, we have an error that appear:
```
Run patrickedqvist/wait-for-vercel-preview@master
  with:
    token: ***
    max_timeout: 60
    check_interval: 2
Error: Cannot read property 'get' of undefined
```

And same for your new version v1.1.0

do you have an idea?

Thanks

**Comment by patrickedqvist** [comment]
@silex-fabrice Thanks for reporting the issue. I think I found the issue and in the new patch hopefully, the issue will be resolved. Give it a go and let me know if you still run into any problems.

**Comment by patrickedqvist** [comment]
This seems to be resolved now (reference #23)
